exercises/bob/README.md

```diff
@@ -6,11 +6,15 @@ Bob answers 'Sure.' if you ask him a question.
 
 He answers 'Whoa, chill out!' if you yell at him.
 
+He answers 'Calm down, I know what I'm doing!' if you yell a question at him.
+
 He says 'Fine. Be that way!' if you address him without actually saying
 anything.
 
 He answers 'Whatever.' to anything else.
 
+Bob's conversational partner is a purist when it comes to written communication and always follows normal rules regarding sentence punctuation in English.
```

Comment:
I think the only thing left is to remove this addition or to just regenerate bob. Since the READMEs are generated, this line will be removed eventually anyway. Did you just write this manually or did it come from somewhere?

Reply:
I wondered about that one as well and would omit it, but it's from the specs.
https://github.com/exercism/problem-specifications/blob/master/exercises/bob/description.md